examples/shader/Shader.cpp

```diff
@@ -307,7 +307,7 @@ class Geometry : public Effect
     void onUpdate(float /*time*/, float x, float y) override
     {
         // Reset our transformation matrix
-        m_transform = sf::Transform::Identity;
+        m_transform = sf::Transform();
```

Comment:
Why these unrelated changes? This makes code arguably less readable.

Reply:
Agreed. Reverted